dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/domain/Account.java

```diff
@@ -44,36 +44,36 @@
    @Nullable
    public abstract String customDefined2();
 
-   public abstract String orgId();
+   @Nullable
+   public abstract String state();
 
    @Nullable
-   public abstract Roles roles();
+   public abstract List<RoleType> roles();
 
-   Account() {
```

Comment:
Why has this been removed? Providing a non-visible default constructor is a best-practice to enforce using the builders. Let's add the constructors back.